### PR TITLE
Adding reviewing process document

### PR DESCRIPTION
**Pull request by arschles**
This document is intended to outline the lightweight process that we’ll use for now. It’s assumed that we’ll operate on good faith for now in situations for which process is not specified.

Also note that this document mentions a `veto` label. It's created [here](https://github.com/kubernetes-incubator/service-catalog/labels/veto)

Makes progress against https://github.com/kubernetes-incubator/service-catalog/issues/50

### CODE REVIEW

File: REVIEWING.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+PRs may only be merged after the following criteria are met:
+
+1. It has been open for at least 1 business day
+1. It has no `veto` label on it
+1. It has been 'LGTM'-ed by 3 different reviewers
+
+## LGTMs
+
+When a reviewer deems a PR ready for review, they should add a comment to the PR
```
Comment (duglin) [inline REVIEWING.md]: I don't think the phrase is "ready for review", I think you mean "ready for merge" or something like that.
And in the next line too.

Comment (arschles) [inline REVIEWING.md]: good catch, thanks @duglin. fixed

**Comment by duglin** [comment]
thanks - LGTM

**Comment by pmorie** [comment]
LGTM

**Comment by duglin** [comment]
actually now that I think about it, can we change it from "veto" to "NOT LGTM". I think that's more consistent with what I see in other repos. Same semantics though.

**Comment by krancour** [comment]
Didn't we also agree that not more than one LGTM can come from reviewers representing a single company?

**Comment by arschles** [comment]
@krancour right you are. I've added that text

File: REVIEWING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+This document is a guideline for how core contributors should review and merge
+pull requests (PRs). It is intended to outline the lightweight process that
+we’ll use for now. It’s assumed that we’ll operate on good faith for now in
+situations for which process is not specified.
+
+PRs may only be merged after the following criteria are met:
+
+1. It has been open for at least 1 business day
+1. It has no `veto` label on it
+1. It has been 'LGTM'-ed by 3 different reviewers, each from a different
```
Comment (bmelville) [inline REVIEWING.md]: I know that we had a bunch of discussion around 3, but I thought that we backed out of that and all decided that 1 non-org LGTM was sufficient for now.

**Comment by krancour** [comment]
LGTM

File: REVIEWING.md (hunk 2 above)
Comment (krancour) [inline REVIEWING.md]: In either case, I believe @pmorie specifically used the phrase "gentle person's agreement--" which I take to mean that we're not formally documenting that bit of leniency.

Comment (bmelville) [inline REVIEWING.md]: SGTM, I must have misremembered the conversation.

**Comment by pmorie** [comment]
Merging.

**Pull request merged**